src/transformers/onnx/features.py

```diff
@@ -351,6 +352,16 @@ class FeaturesManager:
             "token-classification",
             onnx_config_cls=RoFormerOnnxConfig,
         ),
+        "squeezebert": supported_features_mapping(
+            "default",
+            "masked-lm",
+            "causal-lm",
```

Comment:
If I'm not mistaken, SqueezeBert doesn't have a causal LM head, so I suggest removing this feature
```suggestion
```